source/config.py

```diff
@@ -0,0 +1,55 @@
+FAHRENHEIT = "fahrenheit"
+CELSIUS = "celsius"
+METRIC = "metric"
+METERS_PER_SECOND = "meters per second"
+MILES_PER_HOUR = "miles per hour"
+
+
+class WeatherConfig:
+
+    def __init__(self, core_config: dict, settings: dict):
+        self.core_config = core_config
+        self.settings = settings
+        config_location = self.core_config["location"]
+        self.latitude = config_location["coordinate"]["latitude"]
+        self.longitude = config_location["coordinate"]["longitude"]
+        city = config_location["city"]
+        state = city["state"]
+        country = state["country"]
+        self.city = city["name"]
+        self.state = state["name"]
+        self.country = country["name"]
+        self.speed_unit = self._determine_speed_unit()
+        self.temperature_unit = self._determine_temperature_unit()
```

Comment:
Could be an idea to make these properties instead of setting them in `__init__()`, currently I think changes in config/settings affecting these would require reload of the skill but as config / settings are mutated and not recreated a property would update the value automatically.

Reply:
Thanks.  I will look into this.